manifests/apache_config.pp

```diff
@@ -10,15 +10,25 @@
 #
 class php::apache_config(
   Stdlib::Absolutepath $inifile = $php::params::apache_inifile,
-  Hash $settings                = {}
+  Hash $settings                = {},
+  String $service               = nil,
```

Comment:
Could you update this to undef and make it optional? nil is pretty uncommon in Puppet and we try to stick with undef.

Reply:
Thanks for the hint. Should be fixed now.